### PR TITLE
Support GHC 8.8 and 8.10

### PR DESCRIPTION
**Pull request by soareschen**
This change is [Reviewable](https://reviewable.io/reviews/scrive/log/54)

### CODE REVIEW

**Comment by arybczak** [comment]
You have old version of haskell-ci (it didn't add 8.8.3 nor 8.10.1), use the one from git.

Also, let's not test with GHC 7.*.

**Comment by arybczak** [comment]
Also, if repo has multiple packages, `.travis.yml` should be generated using `haskell-ci cabal.project` command.

**Comment by soareschen** [comment]
Fixed

File: log-postgres/log-postgres.cabal [hunk 1]
```diff
@@ -18,16 +18,15 @@ copyright:           Scrive AB
...
+tested-with:         GHC ==8.0.2 || ==8.2.2 || ==8.4.4 || ==8.6.5 || ==8.8.3 || ==8.10.1
 
 Source-repository head
   Type:     git
   Location: https://github.com/scrive/log.git
 
 library
   exposed-modules:    Log.Backend.PostgreSQL
-  build-depends:      base                  >= 4.9  && < 4.12 || ^>= 4.12
+  build-depends:      base                  >= 4.9  && < 5
```
Comment (phadej) [inline log-postgres/log-postgres.cabal:29]: `log-base` still has `base >=4.8` lower bound.

**Comment by soareschen** [comment]
CI tests for GHC 8.8+ are failing because of bloodhound not currently supporting it. I have submitted https://github.com/bitemyapp/bloodhound/pull/269 to fix that.

File: log-postgres/log-postgres.cabal (hunk 1 above)
Comment (soareschen) [reply log-postgres/log-postgres.cabal:29]: Fixed

File: log-postgres/log-postgres.cabal [hunk 2]
```diff
@@ -38,7 +37,7 @@ library
                     , vector               ^>= 0.12
                     , aeson-pretty         ^>= 0.8
                     , base64-bytestring    ^>= 1.0
-                    , hpqtypes             ^>= 1.7  || ^>= 1.8
+                    , hpqtypes             ^>= 1.7  || ^>= 1.9
```
Comment (arybczak) [inline log-postgres/log-postgres.cabal:40]: Shouldn't it be `>=1.7 && <= 2.0`?

**Comment by arybczak** [comment]
I allowed failures for GHC >= 8.8. I'll make a release shortly, log-elasticsearch will work with GHC >= 8.8 out of the box once there's a new release of bloodhound.

**Pull request merged**